tensorboard/webapp/feature_flag/effects/feature_flag_effects_test.ts

```diff
@@ -62,6 +62,7 @@ describe('feature_flag_effects', () => {
     it('loads features from the data source on init', () => {
       spyOn(dataSource, 'getFeatures').and.returnValue({
         enabledExperimentalPlugins: ['foo', 'bar'],
+        inColab: false,
```

Comment:
Perhaps we can use `buildFeatureFlags({enabledExperimentalPlugins: ['foo', 'bar']})` and drop the `inColab` here and below?

Reply:
I will not remove it.

I mean, I am not going to pass every single possible thing but this test spec is neither specifically about the `enabledExperimentalPlugins` or `inColab`. However, exercising both is a valid test imo.

Reply:
This list of files that require all feature flags (selectors.ts, effects_test.ts, testing.ts, types.ts) is small enough that the PR as-is looks reasonable to me.